lib/package.rb

```diff
@@ -129,6 +129,7 @@ def self.system(*args, **opt_args)
     else
       env = CREW_ENV_OPTIONS_HASH
```

Comment:
Should this line be `env = CREW_ENV_OPTIONS`?  No `_HASH`?

Reply:
@supechicken ?

Reply:
no. env needs to be a hash. That's the format that system takes. See https://ruby-doc.org/core-3.0.2/Kernel.html#method-i-system